ApplicationServerDeployment/lambda/alexa/alexa_lambda_handler.py

```diff
@@ -0,0 +1,54 @@
+from discovery import DiscoveryControl
+from power_control import PowerControl
+from report_state import ReportState
+
+from utils import *
+import logging
+import json
+from response import AlexaResponse
+
+logger = logging.getLogger()
+if logger.handlers:
+	for handler in logger.handlers:
+		logger.removeHandler(handler)
+logging.basicConfig(format='%(asctime)s %(message)s', level=logging.ERROR)
+
+def lambda_handler(event, context):

```

Comment:
add block comment to explain the purpose of the function

Reply:
Done